ddtrace/ext/aws.py

```diff
@@ -1,7 +1,10 @@
 from ..utils.formats import flatten_dict
 
 
-BLACKLIST_ENDPOINT = ["kms", "sts"]
+BLACKLIST_ENDPOINT = ['kms', 'sts']
+BLACKLIST_ENDPOINT_TAGS = {
+    's3': ['params.Body'],
```

Comment:
What do you think about just `['params.Body']` and not have it scoped to just `s3`?

Reply:
Planning to add `'cognitoidentity': [....]` as well in this PR.